### ISSUE TITLE
Tutorial stops at chapter 4 "Services"

### ISSUE THREAD

**Issue by martinborgman** [angular/angular.io#660]
The current version of the "Tutorial: The Tour of Heroes" ends at chapter 4 "Services" somewhere halfway the implementation of the Vision.
You can manually go to chapter 5 by changing the URL to https://angular.io/docs/ts/latest/tutorial/toh-pt5.html but this page contains many error messages like:

```
BAD FILENAME: ../../../_fragments/toh-4/ts-snippets/app/app.component.pt4.ts.md   Current path: docs,ts,latest,tutorial,toh-pt5 PathToDocs: ../../../
```

**Comment by wardbell** [comment]
We know! Tutorial is a work in progress. Part 5 is a sketch and not ready to be seen ... which is why didn't link to it.

It will be ready soon. Please hang in there.

Closing this issue for now. Reopen if we don't make good on the promise of part 5 soon.

**Comment by ibufu** [comment]
Where could see the sketch?

**Comment by alisstex** [comment]
Can we reopen this issue please?

**Comment by wardbell** [comment]
Ok. But please ... no more complaints about ToH part 5 being "broken". It was never published so it can't be broken. It does need to get finished.

**Comment by babant** [comment]
Fixed by PR #844.

**Comment by wardbell** [comment]
It's fixed now.
